monitoring/alerts.yaml

```diff
@@ -141,7 +144,9 @@ groups:
   - alert: QuotaMetricsNotAvailable
     expr: |
       avg(avg_over_time(s3_cloudserver_quota_utilization_service_available{namespace="${namespace}",service="${service}"}[1m]))
```

Comment:
what is the point of averaging over the last minute?
we scrap resources every 30 second, so does not add much... and alerts have their own mecanism to smooth over time (i.e. the `for` parameter)

Reply:
Indeed, I:
- Removed the avg_over_time over the last minute
- Removed the `for` for the warning version of this alert